docs/guide/start-installation.md

```diff
@@ -214,8 +214,30 @@ the installed application. You only need to do these once for all.
    127.0.0.1   backend.test
    ```
 
-To login into the application, you need to first sign up, with any of your email address, username and password.
-Then, you can login into the application with same email address and password at any time.
+6. Open your browser and go to http://frontend.test/
+
+7. Create a user by selecting the Sign Up menu option at the top of the frontend home page.
+   
+8. Provide the requested credentials, and complete the data entry with the Signup button. You will be presented with a message:
+   ```
+   Thank you for registration. Please check your inbox for verification email.
+   ``` 
+9. Despite stating that a confirmation email has been sent, the default settings for the mailer prevents the sending of a real email. 
+   Instead, an eml format file is created in the directory `````@frontend/runtime/mail`````. 
```

Comment:
```suggestion
   Instead, an eml format file is created in the directory `@frontend/runtime/mail`. 
```

Reply:
Included in [90b96fb](https://github.com/yiisoft/yii2-app-advanced/pull/516/commits/90b96fb1f3bc6d696499e9abca348bb4f7a7bd38)